packages/plans-grid/src/hooks/use-billing-period.tsx

```diff
@@ -0,0 +1,3 @@
+export default function useBillingPeriod(): 'MONTHLY' | 'ANNUALLY' {
+	return 'MONTHLY';
```

Comment:
At the moment, in Gutenboarding, without any billing period toggle, the plan products rendered in the accordion are monthly.
The default should be annual products.

```suggestion
	return 'ANNUALLY';
```